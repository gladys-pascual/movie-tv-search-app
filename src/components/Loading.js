import React from "react";
import PacmanLoader from "react-spinners/PacmanLoader";

const Loading = () => {
  return (
    // <section className="loading">
    //   <span className="material-icons">hourglass_bottom</span>
    //   <h3>Loading...</h3>
    // </section>

    <div className="loading">
      <PacmanLoader color="#7bcbc4" size="100" />
    </div>
  );
};

export default Loading;
