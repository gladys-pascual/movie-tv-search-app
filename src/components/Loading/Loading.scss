.loading {
  padding-top: 20vh;
  display: flex;
  justify-content: center;
  max-width: 100%;
}
