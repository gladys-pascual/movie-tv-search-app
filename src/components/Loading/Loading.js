import React from "react";
import PacmanLoader from "react-spinners/PacmanLoader";
import "./Loading.scss";

const Loading = () => {
  return (
    <div className="loading">
      <PacmanLoader color="#7bcbc4" size="60px" />
    </div>
  );
};

export default Loading;
