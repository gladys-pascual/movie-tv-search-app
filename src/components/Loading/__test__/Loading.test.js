import React from "react";
import { render } from "@testing-library/react";
import Loading from "../Loading";

test("renders Loading component as expected", () => {
  const { container } = render(<Loading />);
  expect(container).toMatchSnapshot();
});
