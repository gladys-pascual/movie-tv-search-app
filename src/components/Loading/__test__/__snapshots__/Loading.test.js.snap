// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Loading component as expected 1`] = `
<div>
  <div
    class="loading"
  >
    <div
      class="css-8pkz"
    >
      <div
        class="css-rxlor6"
      />
      <div
        class="css-13u5b26"
      />
      <div
        class="css-144dyr7"
      />
      <div
        class="css-pu3zi4"
      />
      <div
        class="css-tzx4d9"
      />
      <div
        class="css-1mjpbtg"
      />
    </div>
  </div>
</div>
`;
